src/vtquery.cpp

```diff
@@ -6,14 +6,39 @@
 #include <iostream>
 #include <map>
 #include <stdexcept>
-// #include <mapbox/geometry/geometry.hpp>
-// #include <mapbox/geometry/algorithms/closest_point.hpp>
+#include <mapbox/geometry/geometry.hpp>
+#include <mapbox/geometry/algorithms/closest_point.hpp>
 #include <vtzero/vector_tile.hpp>
 #include <vtzero/types.hpp>
-// #include <mapbox/variant.hpp>
+#include <mapbox/variant.hpp>
+
+#include "geometry_processors.hpp"
+#include "reproject.hpp"
 
 namespace VectorTileQuery {
 
+struct ResultObject {
+    ResultObject(
+        std::uint32_t z0,
+        std::uint32_t x0,
+        std::uint32_t y0,
+        mapbox::geometry::algorithms::closest_point_info cp_info0,
```

Comment:
@mapsam: noticed this will need to be `mapbox::geometry::algorithms::closest_point_info<std::int64_t>` I think since `closest_point_info` is templated on coordinate type.